CLAUDE.md

```
@@ -0,0 +1,393 @@
+- **Document Parsing**: PyPDF2, python-docx, pdfplumber, Tesseract OCR
+- **NLP**: spaCy (legal entity recognition), NLTK (text processing)
+
+### Development Tools
+- **Testing**: pytest, unittest, hypothesis (property-based testing)
+- **Code Quality**: black, ruff, mypy, pylint
+- **CI/CD**: GitHub Actions
+- **Documentation**: MkDocs, Sphinx
+
+## Repository Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)